ui/plugin-system/src/runtime/time-range.ts

```diff
@@ -12,11 +12,13 @@
 // limitations under the License.
 
 import { createContext, useContext } from 'react';
-import { AbsoluteTimeRange, DurationString } from '@perses-dev/core';
+import { AbsoluteTimeRange, TimeRangeValue } from '@perses-dev/core';
 
 export interface TimeRange {
-  timeRange: AbsoluteTimeRange;
-  defaultDuration: DurationString;
+  timeRange: TimeRangeValue;
+  absoluteTimeRange: AbsoluteTimeRange;
+  initialValue: TimeRangeValue;
```

Comment:
Is `initialValue` here just for resetting/undo of some kind? If so, would it be better to just expose a `reset` API rather than expose this?

Reply:
yeah I was thinking we might need it later for undo / restoring to original state, but a reset API makes more sense so just removed it